### ISSUE TITLE
'about' and 'frequently asked questions'

### ISSUE THREAD

**Issue by indirectlylit** [fle-internal/kolibri-instant-schools-plugin#56]
Notes:

* When a user logs in using [a newly-created profile](https://github.com/fle-internal/kolibri-instant-schools-plugin/issues/55) they should be forwarded to the 'learn' page. Otherwise they should be redirected to 'learn'
* Content for both pages should be editable _after the PEX has been built_. This means that the HTML and images need to be loaded through Django's static file server, not bundled into webpack.

-----

New link in Nav:

![image](https://user-images.githubusercontent.com/2367265/31692122-5f257e7c-b34d-11e7-809c-ba317d55419f.png)

'About' page with its own URL:

![image](https://user-images.githubusercontent.com/2367265/31692129-6894ea7e-b34d-11e7-9cd0-1d7d9e9a943e.png)

FAQ page with its own URL:

![image](https://user-images.githubusercontent.com/2367265/31692139-72ec79c4-b34d-11e7-8238-878cf8120444.png)

**Comment by christianmemije** [comment]
So these would be iframes?

**Comment by indirectlylit** [comment]
That is definitely a possibility and maybe the easiest strategy if it's possible to get the layout right.

It would also be possible to render inline using `v-html`. The iframe would better isolate the HTML, which prevents the opcos from breaking the rest of Kolibri. However, I don't think that is a huge risk.

**Comment by christianmemije** [comment]
## Brainstorming

```
/user#/about
/user#/faq
```

* 2 vue components with iframes pointing the static html files

```
views/about/index.vue
views/about/faq.vue
views/about/goodName/about.html
views/about/goodName/faq.html
views/about/goodName/assets/logo.png
views/about/goodName/assets/minimalstyle.css
etc...
```

* Get started and start learning links will be within the vue components not the iframes
* html is barebones

Static files

```
.kolibri/content/about/about.html
.kolibri/content/about/faq.html
.kolibri/content/about/assets/logo.png
.kolibri/content/about/assets/minimalstyle.css
```

## Questions

* Where to add new static dir url in python?
* How to copy contents of `goodName` dir to `.kolibri/content/about`?
* Upgrade/update overwriting files?

cc @rtibbles @indirectlylit

**Comment by indirectlylit** [comment]
quick note - haven't read through in detail yet, but:

> /user#/about
> /user#/faq

@rtibbles had a PR to give these routes on the server-side: https://github.com/fle-internal/kolibri-instant-schools-plugin/pull/47/files

My preference would actually be a server route for `/about` and a client route `for `/about/#/faq`

**Comment by indirectlylit** [comment]
> Get started and start learning links will be within the vue components not the iframes

Yes - I'd forgotten to mention this, but that was the plan so we could re-use `k-button` components, etc

**Comment by christianmemije** [comment]
I'd prefer iframe over v-html since it is much more isolated. If we use v-html we would have to pass in the html and hope that the assets are referenced correctly. the iframe route might make it easier for the admin to change and test the html.

**Comment by christianmemije** [comment]
Should routing to the about page should happen on new profile sign in, new account sign in, or sign up?
 @jamalex

**Comment by jamalex** [comment]
Most of the above. Specifically:

- New profile sign-in (selection of a profile that was just created)
- Post-signup for totally new accounts (it auto-logs in at that point)
- Anonymous users when they click "access as guest", if they don't have a
special cookie set. And then set the cookie.

**Comment by christianmemije** [comment]
Implemented in #61 . What remains is to redirect on new profile sign in. An issue for this was opened: https://github.com/fle-internal/kolibri-instant-schools-plugin/issues/76
